src/jio-navbar.css

```diff
@@ -208,6 +208,11 @@ button.nav-link {
     order: 5;
   }
 
+  .dropdown{
```

Comment:
Can you add a space between selector and bracket to match the rest? I'm surprised there's no linting error.

Reply:
@halkeye Done.